modules/lvms-reference-file.adoc

```diff
@@ -23,54 +23,101 @@ spec:
         optionalPaths:
         - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
         - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
-        forceWipeDevicesAndDestroyAllData: true <6>
-      thinPoolConfig: <7>
-        name: thin-pool-1 <8>
-        sizePercent: 90 <9>
-        overprovisionRatio: 10 <10>
+        forceWipeDevicesAndDestroyAllData: true 
+      thinPoolConfig: <4>
+        name: thin-pool-1 
+        sizePercent: 90 
+        overprovisionRatio: 10 
 status:
-    deviceClassStatuses: <11>
+    deviceClassStatuses: <5>
     - name: vg1
-      nodeStatus: <12>
-      - devices: <13>
+      nodeStatus: 
+      - devices: 
         - /dev/nvme0n1
         - /dev/nvme1n1
         - /dev/nvme2n1
-        node: my-node.example.com <14>
-        status: Ready <15>
-    ready: true <16>
-    state: Ready <17>
+        node: my-node.example.com 
+        status: Ready 
+    ready: true 
+    state: Ready <6>
 ----
-<1> The LVM volume groups to be created on the cluster. Currently, only a single `deviceClass` is supported.
-<2> The name of the LVM volume group to be created on the nodes.
-<3> The nodes on which to create the LVM volume group. If the field is empty, all nodes are considered.
-<4> A list of node selector requirements.
-<5> A list of device paths which is used to create the LVM volume group. If this field is empty, all unused disks on the node will be used.
-<6> To force wipe the selected disks, set this parameter to `true`. This parameter is set to `false` by default.
-<7> The LVM thin pool configuration.
-<8> The name of the thin pool to be created in the LVM volume group.
-<9> The percentage of remaining space in the LVM volume group that should be used for creating the thin pool.
-<10> The factor by which additional storage can be provisioned compared to the available storage in the thin pool.
-<11> The status of the `deviceClass`.
-<12> The status of the LVM volume group on each node.
-<13> The list of devices used to create the LVM volume group.
-<14> The node on which the `deviceClass` was created.
-<15> The status of the LVM volume group on the node.
-<16> This field is deprecated.
-<17> The status of the `LVMCluster`.
+<1> `deviceClasses`: Contains the configuration to assign the local storage devices to the LVM volume groups. To create multiple device storage classes in the LVM cluster, create an array under `deviceClasses` for each storage class.
++
+[NOTE]
+====
+If you add or remove a `deviceClass`, then the update reflects in the LVM cluster only after deleting and recreating the `TopoLVM-Node` pod. 
+====
++
+The `deviceClasses` contains the following fields:
+
+`name`:: Specify the name of the `deviceClass`.
+`fstype`:: Set the file system to `ext4` or `xfs`. By default, it is set to `xfs`.
+`default`:: You can set this field to `true` to indicate that a `deviceClass` is the default. Else, you can set it to `false`. You can configure only a single default `deviceClass`. 
```

Comment:
See my comment above about combining code callouts and description lists. 

I suggest reworking this content into separate concept modules about architecture or workflows and maybe another reference module as a table describing the fields.